recipes/libcroco/build.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+# Copyright 2016 Peter Williams and collaborators.
+# This file is licensed under a 3-clause BSD license; see LICENSE.txt.
+
+[ "$NJOBS" = '<UNDEFINED>' -o -z "$NJOBS" ] && NJOBS=1
+set -e
+test $(echo "$PREFIX" |wc -c) -gt 200 # check that we're getting long paths
+
+# don't get locally installed pkg-config entries:
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig:$PREFIX/share/pkgconfig"
```

Comment:
The copyright lines and everything up until this `export` line should be removed. They're specific to my build system but aren't what we want on conda-forge.